rasa/nlu/test.py

```diff
@@ -1050,12 +1051,24 @@ def get_eval_data(
                 response_prediction_key, {}
             ).get(OPEN_UTTERANCE_PREDICTION_KEY, {})
 
+            response_prediction_full_intent = selector_properties.get(
+                response_prediction_key, {}
+            ).get("full_retrieval_intent", {})
+
+            if isinstance(response_prediction_full_intent, str):
+                response_prediction_full_intent = response_prediction_full_intent.split(
+                    "/"
+                )[1]
+
             response_target = example.get("response", "")
+            response_key = example.get(RESPONSE_KEY_ATTRIBUTE, "")
```

Comment:
Following from the above comment, this would then change to `example.get_combined_intent_response_key()`